tests/ignite/contrib/handlers/test_lr_finder.py

```diff
@@ -230,6 +230,10 @@ def test_plot(lr_finder, to_save, dummy_engine, dataloader):
 
     lr_finder.plot()
     lr_finder.plot(skip_end=0)
+    lr_finder.plot(skip_end=0, filepath="dummy.jpg")
+    lr_finder.plot(
+        skip_end=0, filepath="dummy.jpg", orientation="landscape", papertype="a4", format="png",
```

Comment:
We do not check if file exists, right ? Can you please add this test ?

Reply:
I've added the test as well as the extra try/check. That's really good catch. Thanks a lot

Reply:
Actually, my comment was about the test:
```python
lr_finder.plot(skip_end=0, filepath="dummy.jpg")
assert Path("dummy.jpg").exists()
```
However, I agree that if we run it like : 
```python
lr_finder.plot(skip_end=0, filepath="/tmp/nonexisting-folder/dummy.jpg")
assert Path("/tmp/nonexisting-folder/dummy.jpg").exists()
```
we have to ensure parents folders creation as well.